docs/lmstudio.md

```diff
@@ -6,7 +6,7 @@
 
     If you see "Prompt Formatting" (inside LM Studio's "Server Options" panel), turn it **OFF**. Leaving it **ON** will break MemGPT.
 
-![image](https://github.com/cpacker/MemGPT/assets/5475622/abc8ce2d-4130-4c51-8169-83e682db625d)
+![LMS PF](https://github.com/cpacker/MemGPT/assets/65172063/673cbd17-31eb-4292-8021-b6010894bc36)
```

Comment:
awesome, ty! do you think you could update this screenshot to also show / highlight the context length option?

(since that's the other important option outlined here: https://memgpt.readthedocs.io/en/latest/lmstudio/)

Reply:
also, do you think you could update the screenshot to use the same dolphin model? just to keep the screenshot consistent with what's being recommended in the readthedocs

Reply:
Sure:

![image](https://github.com/cpacker/MemGPT/assets/65172063/a34f358e-9d77-4b30-86ff-36847044774a)